.babelrc

```diff
@@ -1,50 +1,88 @@
 {
   "env": {
-    "test": {
+    "development": {
       "presets": [
-        ["env", {"targets": { "node": "current" }}],
+        [
+          "env",
+          {
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 9"]
+            }
+          }
+        ],
         "react",
         "stage-2"
-      ],
-      "plugins": [
-        "babel-plugin-rewire",
-        ["module-resolver", {
-          "alias": {
-            "preact-compat": "react"
+      ]
+    },
+    "production": {
+      "presets": [
+        [
+          "env",
+          {
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 9"]
+            }
           }
-        }]
+        ],
+        "react",
+        "stage-2"
       ]
     },
-    "development": {
+    "umd": {
       "presets": [
-        ["env", { "targets": { "browsers": ["last 2 versions", "ie >= 10"] } }],
+        [
+          "env",
+          {
+            "modules": false,
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 9"]
+            }
+          }
+        ],
         "react",
         "stage-2"
       ]
     },
-    "production": {
+    "es": {
       "presets": [
-        ["env", { "targets": { "browsers": ["last 2 versions", "ie >= 10"] } }],
+        [
+          "env",
+          {
+            "modules": false,
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 9"]
+            }
+          }
+        ],
         "react",
         "stage-2"
       ],
       "plugins": [
-        "transform-react-remove-prop-types",
-        "transform-react-constant-elements",
-        "transform-react-pure-class-to-function"
```

Comment:
Why did we remove those?

Reply:
I added them to the Rollup configuration instead. But you're right, we need them in CJS and ES too.